editors/emacs/Portfile

```diff
@@ -54,24 +54,29 @@ configure.args  --disable-silent-rules \
                 --without-tree-sitter \
                 --with-libgmp \
                 --with-gnutls \
-                --with-json \
                 --with-xml2 \
                 --with-modules \
                 --with-sqlite3 \
                 --with-webp \
+                --with-native-compilation=no \
                 --infodir ${prefix}/share/info/${name}
 
+# Later-on this could potentially be a variant, given enough demand. See:
+# https://github.com/emacs-mirror/emacs/commit/4c6f45fa8eef1a15d5790c1f3d3e608b548015db#diff-731c852b7c27c6ee90c44c871e2988ecfc2d225bc720156d7a47c1d563c2683aR59
+configure.args-append  --disable-gc-mark-trace
+
 pre-configure {
     if {[llength $rpaths] > 0} {
         configure.ldflags-append "-Wl,-rpath " [join $rpaths " -Wl,-rpath "]
     }
+
+    system -W ${worksrcpath} "sh ./autogen.sh"
```

Comment:
Is this how this is done? Is there not some `use_autogen yes` whatever?

Reply:
There's a `use_autoreconf yes`, but nothing for autogen.sh specifically.